shuup/admin/templates/shuup/admin/auth/login.jinja

```diff
@@ -40,10 +40,17 @@
 {% endblock %}
 {% block extra_js %}
 <script>
+
 $(function () {
     var fields = $("input");
 
     fields.each(function() {
+        if ($(this).val().length > 0) {
+                $('.btn-primary').removeClass('disabled');
+        } else {
+            $('.btn-primary').addClass('disabled');
+        } // DRY Perhaps this could be turned into a function?
```

Comment:
Yes looks like duplicate code and yes you should likely clean this up a little bit. Also please do not introduce any new TODOs in the small fixing tasks. Point of these small tasks is to actually deal with these kind of small TODOs we have left behind.